pkgs/applications/networking/instant-messengers/riot/riot-desktop.nix

```diff
@@ -1,7 +1,4 @@
-{ stdenv, fetchFromGitHub
-, makeWrapper, makeDesktopItem, mkYarnPackage
-, electron_7, riot-web, gtk3,
-}:
+{ pkgs, stdenv, fetchFromGitHub, makeWrapper, makeDesktopItem, electron_7, riot-web, mkYarnPackage, wrapGAppsHook, glib, gtk3 }:
```

Comment:
I don't really like this: it makes the diff really hard to read. :(

Reply:
you can diff on github without whitespace changes.
and I realized I didn't intend to change this.